docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  calitp_reports:
+    container_name: calitp-reports
+    build:
+      context: .
+      dockerfile: config/docker/Dockerfile
+    volumes:
+      - $HOME/.config/gcloud:/home/reports/.config/gcloud
+      - $PWD/reports:/home/reports/app
+    ports:
+      # To run container jupyter notebooks on the host
+      - 8891:8891
+    environment:
+      - GOOGLE_APPLICATION_CREDENTIALS=/home/reports/.config/gcloud/service-account-file.json
+      # Set the service key path for calitp-py
+      - CALITP_SERVICE_KEY_PATH=/home/reports/.config/gcloud/service-account-file.json
```

Comment:
Notice these are the same values, should `GOOGLE_APPLICATION_CREDENTIALS` be the `application_default_credentials.json` file?

Reply:
They point to the same thing as a service account. That variable isn't needed though, and can be removed.